openreview/venue/invitation.py

```diff
@@ -3095,4 +3095,100 @@ def set_SAC_ethics_flag_invitation(self, sac_ethics_flag_duedate=None):
             if sac_ethics_flag_duedate:
                 invitation.edit['invitation']['expdate'] = tools.datetime_millis(sac_ethics_flag_duedate)
 
-            self.save_invitation(invitation, replacement=True)
\ No newline at end of file
+            self.save_invitation(invitation, replacement=True)
+
+    def set_group_recruitment_invitations(self, committee_name):
+
+        venue_id = self.venue_id
+        venue = self.venue
+
+        invitation = Invitation(id=venue.get_group_recruitment_id(committee_name),
+            invitees=[venue_id],
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=['~Super_User1'],
+            process=self.get_process_content('process/group_recruitment_process.py'),
+            content={
+                'committee_name': { 'value': committee_name },
+                'official_committee_roles': { 'value': venue.get_committee_names()}
+            },
+            edit={
+                'signatures': [venue_id],
+                'readers': [venue_id],
+                'writers': [venue_id],
+                'content': {
```

Comment:
@melisabok I am adding the different recruitment status to the content here, so I can add them to the recruitment edit. However, how can we differentiate between the fields that the PCs have to fill out (`inviteeDetails`) and the ones I am using to store other things (like the recruitment status)

Reply:
can recruitment status be hidden in the note editor?